docs/docs/training-data-format.mdx

```diff
@@ -428,6 +428,42 @@ policies learn to predict the next action based on a *combination* of
 both the intent and entities (you can, however, change this behavior
 using the [`use_entities`](#entities) attribute).
 
+If you use `user:` key while writing stories, you do not have to deal with the specific
+intents of the messages that are extracted by the NLU pipeline. Instead, you can simply put
+the text of the user message.
+We call it end-to-end training because actual text is used for predictions.
+
+:::caution
+This feature is experimental.
+We introduce experimental features to get feedback from our community, so we encourage you to try it out!
+However, the functionality might be changed or removed in the future.
+If you have feedback (positive or negative) please share it with us on the [Rasa Forum](https://forum.rasa.com).
+
+:::
+
+These end-to-end user messages follow the format:
+
+```yaml-rasa
+stories:
+- story: user message structure
+  steps:
+    - user: the actual text of the user message  # Required
+```
+
+In addition, you can add entity tags that will be extracted by the end-to-end policy.
+The syntax for entity tags is the same as in
+[the NLU training data](./training-data-format.mdx#entities)
+For example, the following story contains the user utterance
+`I want to check my credit balance`. By using the syntax from the NLU training data
+`[credit](account_type)`, we mark `credit` as an entity of type `account_type`.
```

Comment:
Not an expert on doc guidelines, but according to [this](https://www.notion.so/rasa/Tone-and-language-guidelines-cd96f21292b24c1ab7f7457c7973ec69#766266eaf5744ca094bd247409684d28) we should say `you` (#soMeta)

Reply:
where?